_code-samples/issue-mpt-with-metadata/js/issue-mpt-with-metadata.js

```diff
@@ -1,85 +1,108 @@
-import { stringToHex, hexToString } from '@xrplf/isomorphic/dist/utils/index.js'
-import { MPTokenIssuanceCreateFlags, Client } from 'xrpl'
+import {
+  MPTokenIssuanceCreateFlags,
+  Client,
+  encodeMPTokenMetadata,
+  decodeMPTokenMetadata,
+} from "xrpl";
 
 // Connect to network and get a wallet
-const client = new Client('wss://s.devnet.rippletest.net:51233')
-await client.connect()
+const client = new Client("wss://s.devnet.rippletest.net:51233");
+await client.connect();
 
-console.log('Funding new wallet from faucet...')
-const { wallet } = await client.fundWallet()
+console.log("=== Funding new wallet from faucet...===");
+const { wallet: issuer } = await client.fundWallet();
+console.log(`Issuer address: ${issuer.address}`);
 
 // Define metadata as JSON
 const mpt_metadata = {
```

Comment:
```suggestion
const mptMetadata = {
```
We should keep to camelcase for javascript code. This applies to all subsequent snake cases.